group/templates/group/groupDetail.html

```diff
@@ -45,196 +48,262 @@ <h3 style="display:inline">,&nbsp</h3>
+      </div>
+      <form action="#" class="navbar-search pull-left" method="get">
+        <input type="text" name="q" class="search-query span10" value=""
+               placeholder="Search..."style="float:left">
+      </form>
+      <div class="btn-group" style="float:right">
+        <a href="#" class="btn btn-default btn-sm">+</a>
+        <a href="#" class="btn btn-defalut dropdown-toggle btn-sm" data-toggle="dropdown">
+          <span class="caret"></span>
+        </a>
+        <ul class="dropdown-menu">
+          <li><a href="#">Add single student</a>
+          </li>
+          <li><a href="#">Add with list</a>
+          </li>
+          <li class="divider"></li>
+          <li><a href="#">Something else</a>
+          </li>
+        </ul>
+        <a href="" class="btn btn-sm btn-default" data-toggle="modal"
+           data-target="#studentDataDownloadModal" style="float:right">Download</a>
+      </div>
+      -->
+      <table class="table table-hover">
+        <thead>
+          <tr>
+            <div>
+              <th>Name</th>
+              <th></th>
             </div>
-          </td>
-        </tr>
-        {% endfor %}
-      </tbody>
-    </table>
-    <a class="btn btn-sm btn-default" style="float:right"
-       onclick="window.open('{% url 'group:viewall_announce' group_id %}', '', config='height=500,width=800');">view all</a>
-    <br>
-  </div>
-  <hr>
-  <div class="member">
-    <form action="#" class="navbar-search pull-left" method="get">
-      <input type="text" name="q" class="search-query span10" value=""
-             placeholder="Search..."style="float:left">
-    </form>
-    <div class="btn-group" style="float:right">
-      <a href="#" class="btn btn-default btn-sm">+</a>
-      <a href="#" class="btn btn-defalut dropdown-toggle btn-sm" data-toggle="dropdown">
-        <span class="caret"></span>
-      </a>
-      <ul class="dropdown-menu">
-        <li><a href="#">Add single student</a>
-        </li>
-        <li><a href="#">Add with list</a>
-        </li>
-        <li class="divider"></li>
-        <li><a href="#">Something else</a>
-        </li>
+          </tr>
+        </thead>
+        <tbody>
+          {% for student in student_list %}
+          <tr>
+            <div>
+              <td>{{ student.username }}</td>
+              <td>
+                {% if user_is_owner or user_is_coowner or user_has_admin_auth %}
+                  <a href="{% url 'group:delete_member' group_id student.username %}">
+                    <button type="button" class="btn btn-danger btn-xs" style="float:right">
+                      <span class="glyphicon glyphicon-remove"></span>
+                    </button>
+                  </a>
+                {% endif %}
+              </td>
+            </div>
+          </tr>
+          {% endfor %}
+        </tbody>
+      </table>
+    </div>
+    <hr>
+    <div class="container-fluid">
+      <ul class="pager">
```

Comment:
You can include ```pager.html``` in [utils](https://github.com/bruce3557/NTHUOJ_web/blob/dev/utils/templates/utils/pager.html)